frontend/src/metabase/query_builder/components/chart-type-selector/ChartTypeList/ChartTypeList.tsx

```diff
@@ -0,0 +1,29 @@
+import { ChartTypeOption, type ChartTypeOptionProps } from "../ChartTypeOption";
+
+import { OptionList } from "./ChartTypeList.styled";
+
+export type ChartTypeListProps = {
+  visualizationList: ChartTypeOptionProps["visualizationType"][];
+  "data-testid"?: string;
+} & Pick<
+  ChartTypeOptionProps,
+  "selectedVisualization" | "onSelectVisualization"
+>;
+
+export const ChartTypeList = ({
+  visualizationList,
+  onSelectVisualization,
+  selectedVisualization,
+  "data-testid": dataTestId,
```

Comment:
Does react recognize `data-*` data attributes as a prop? I have never tried that before.

Reply:
yep! it's just an object so we can do things like this 😄 I think `key` and `ref` are the only reserved props. everything else is fair game